packages/core/src/textures/resources/SVGResource.js

```diff
@@ -115,9 +131,27 @@ export default class SVGResource extends BaseImageResource
                 throw new Error('The SVG image must have width and height defined (in pixels), canvas API needs them.');
             }
 
-            // Scale realWidth and realHeight
-            const width = Math.round(svgWidth * this.scale);
-            const height = Math.round(svgHeight * this.scale);
+            // Set render size
+            let width = svgWidth * this.scale;
+            let height = svgHeight * this.scale;
+
+            if (this._overrideWidth && this._overrideHeight)
```

Comment:
What about this as a more terse version?
```suggestion
            if (this._overrideWidth || this._overrideHeight)
            {
                width = this._overrideWidth || this._overrideHeight / svgHeight * svgWidth;
                height = this_overrideHeight || this._overrideWidth / svgWidth * svgHeight;
            }
```

Reply:
More terse